e2e/testsuite/grpc_query.go

```diff
@@ -285,7 +285,7 @@ func (s *E2ETestSuite) GetBlockHeaderByHeight(ctx context.Context, chain ibc.Cha
 	if res.SdkBlock != nil {
 		return &res.SdkBlock.Header, nil
 	}
-	return &res.SdkBlock.Header, nil
+	return &res.Block.Header, nil // needed for v4 (uses SDK v0.45)
```

Comment:
Should we be using `SdkBlock` when v4 reaches EOL?

Reply:
We should just do `return &res.SdkBlock.Header, nil`, no need to check `SdkBlock` is not nil. I will do this as part of the chores I do when v4 becomes obsolete.